library/src/main/java/net/sharewire/googlemapsclustering/ClusterManager.java

```diff
@@ -38,6 +39,8 @@
 
     private AsyncTask mClusterTask;
 
+    private int mMinClusterSize = 1;
```

Comment:
I prefer to have a constant `DEFAULT_MIN_CLUSTER_SIZE = 1` and then `private int mMinClusterSize = DEFAULT_MIN_CLUSTER_SIZE`.